src/EnergyPlus/InternalHeatGains.cc

```diff
@@ -6660,7 +6660,7 @@ namespace InternalHeatGains {
                                     OutputProcessor::SOVStoreType::Average,
                                     state.dataHeatBal->space(spaceNum).Name);
                 // Not applicable for space until space has it's own air temeratures
-                // SetupOutputVariable(state,
+                // Setup Output Variable(state,
```

Comment:
The SetupOutputVariable parser was warning that it found a commented SetupOutputVariable call, which can cause parsing issues.  I just added spaces here so it wouldn't attempt to parse this.